src/game/puppyprint.h

```diff
@@ -119,4 +120,6 @@ extern char consoleLogTable[LOG_BUFFER_SIZE][255];
 extern void profiler_offset(u32 *time, OSTime time2);
 extern void puppyprint_update_rsp(u8 flags);
 extern void profiler_add(u32 *time, OSTime time2);
+extern void print_small_text_buffered(s32 x, s32 y, const char *str, u8 align, s32 amount, u8 font);
+extern void puppyprint_print_deferred(void);
```

Comment:
Doesn't this make more sense to call internally?

Reply:
The idea being that you can call `puppyprint_print_deferred` from anywhere so you can pop the text stack, just like you can wish the `print_text_symbols` or whatever it's called.

Reply:
Placing a deferred print right next to debug still wouldn't hurt. There's no reason you couldn't call that twice, right?